src/main/java/UseCases/DataRetrieval/CurrentUserGateway.java

```diff
@@ -11,24 +11,12 @@
 
 public class CurrentUserGateway {
```

Comment:
Just wanted to point out that this class should be within the InterfaceAdapters package - even though it is within the Interface Adapters layer, it makes sense for that to reflect within our package structure too as we're structuring and organizing our classes in packages that correspond to each layer.